### PR TITLE
update groundedness prompt

### PR DESCRIPTION
**Pull request by bpmcgough**
TL;DR I was having issues with the groundedness feedback function when passing an object in as the source, and I determined the issue was that the instructions specify to return the "exact sentence" in the source where the data comes from. If the source is an object, there may be no sentence at all, so this was returning nothing. 

After making this change, my groundedness function works as expected, as shown in the video below.

Video walkthrough
https://www.loom.com/share/afdb7a3ac28e4bed9f31458804c357e1?sid=8a310c28-361d-4df4-8fd3-920c87236992

Could be related to https://github.com/truera/trulens/issues/1101

### CODE REVIEW

**Comment by joshreini1** [comment]
LGTM - will run some testing later this week to confirm. Thanks for the change and explanation!

**Review by joshreini1 — APPROVED** [review]
